electrum/simple_config.py

```diff
@@ -614,7 +614,12 @@ def __setattr__(self, name, value):
 
     # config variables ----->
     NETWORK_AUTO_CONNECT = ConfigVar('auto_connect', default=True, type_=bool)
-    NETWORK_ONESERVER = ConfigVar('oneserver', default=False, type_=bool)
+    NETWORK_ONESERVER = ConfigVar(
+        'oneserver', default=False, type_=bool,
+        short_desc=lambda: _('Connect only to a single Electrum Server'),
+        long_desc=lambda: _('Connect only to a single Electrum Server. This is only intended for connecting to your own node.') + ' ' +
+                          _('Using this option on a public server is a security risk and is discouraged.')
```

Comment:
When splitting messages for localization, please have a think about the work of translators.
 - "Connect only to a single Electrum Server" will be translated twice.
 - "This is only intended for connecting to your own node" and "Using this option on a public server is a security risk and is discouraged" are provided as two separate messages, meaning translators will not have the context of the first message when translating the second.